src/main/scala/io/viash/functionality/dependencies/Repository.scala

```diff
@@ -73,6 +73,12 @@ object Repository {
           repo = repo,
           tag = getGitTag(tag)
         ))
+      case sugarSyntaxRegex("viashhub" | "viash-hub" | "viash", repo, tag) =>
```

Comment:
I feel like we should decide on just one of these protocol identifiers :P

```suggestion
      case sugarSyntaxRegex("viashhub" | "viash-hub" | "viash" | "vhub" | "vsh" | "vhb", repo, tag) =>
```